packages/create-mc-app/src/cli.ts

```diff
@@ -70,6 +70,10 @@ const run = () => {
       '--initial-project-key <value>',
       '(optional) A commercetools project key used for the initial login in development. By default, the value is prompted in the terminal.'
     )
+    .option(
+      '--cloud-identifier <value>',
+      '(optional) The cloud region for the new project. (default: gcp-eu)'
```

Comment:
One last thing, `project` holds a [specific meaning in our glossary](https://docs.commercetools.com/merchant-center/projects).
```suggestion
      '(optional) Cloud region identifier (default: gcp-eu)'
```